packages/auto-approve/src/utils-for-pr-checking.ts

```diff
@@ -118,17 +125,31 @@ export function getVersions(
 
   const oldVersions = versionFile.patch?.match(oldVersionRegex);
   const newVersions = versionFile.patch?.match(newVersionRegex);
-
   if (oldVersions) {
-    oldDependencyName = oldVersions[1];
-    oldMajorVersion = oldVersions[2];
-    oldMinorVersion = oldVersions[3];
+    if (process === 'java-dependency') {
+      oldDependencyName = `${oldVersions[1]}:${oldVersions[2]}`;
+      oldMajorVersion = oldVersions[4] || oldVersions[6];
+      oldMinorVersion = oldVersions[5] || oldVersions[7];
+    } else {
+      oldDependencyName = oldVersions[1];
+      oldMajorVersion = oldVersions[2];
+      oldMinorVersion = oldVersions[3];
+    }
   }
 
   if (newVersions) {
-    newDependencyName = newVersions[1];
-    newMajorVersion = newVersions[2];
-    newMinorVersion = newVersions[3];
+    if (process === 'java-dependency') {
+      newDependencyName = `${newVersions[1]}:${newVersions[2]}`;
+      newMajorVersion = newVersions[5] || newVersions[7];
+      newMinorVersion = newVersions[6] || newVersions[8];
+      console.log(newDependencyName);
+      console.log(newMajorVersion);
+      console.log(newMinorVersion);
```

Comment:
Did you mean to include `console.log()` here?

Reply:
Good catch, thanks!